modules/gapi/test/cpu/gapi_ocv_stateful_kernel_tests.cpp

```diff
@@ -51,8 +51,6 @@ namespace
         {
             // Since G-API has no own test data (yet), it is taken from the common space
             const char* testDataPath = getenv("OPENCV_TEST_DATA_PATH");
-            GAPI_Assert(testDataPath != nullptr);
-
             cvtest::addDataSearchPath(testDataPath);
```

Comment:
Assertions are usually used to check assumptions of the code below.

So with removal of assertion you must check/fix the code below:

```
if (testDataPath)
    cvtest::addDataSearchPath(testDataPath);
```

Reply:
Ok.

Reply:
Fixed.